app_dart/lib/src/service/luci_build_service.dart

```diff
@@ -53,6 +58,9 @@ class LuciBuildService {
   static const int kDefaultPriority = 30;
   static const int kRerunPriority = 29;
 
+  /// Name of the subcache to store luci build related values in redis.
+  static const String subCacheName = 'luci';
```

Comment:
It looks like the cache size for in-memory is 256. I am not sure if that is sub-caches or if this is total elements. How much will doing this affect the cache size?

Reply:
Are you referring to `inMemoryMaxNumberEntries`? This is for `inMemory = true`. For our production server, they are using redis cache. Also this number is for number of entries, and we are consuming one entry for this purpose. Even with inMemory, we should have enough entries left.